polyjuice/dicom_image.py

```diff
@@ -1,14 +1,24 @@
 import dicom
-
+import os
 class DicomImage(object):
 
     def __init__(self, dicom_file):
         self._dataset = dicom.read_file(dicom_file)
         self.filepath = dicom_file.name
 
+    def get_metadata(self, metadata_output):
+        _dataset = self._dataset
+        patient_id = self.get_patient_id()
+        json_file_name = patient_id+".txt"
+        json_file = os.path.join(metadata_output, json_file_name)
```

Comment:
varibable naming. This isnt a file, its a path. Also it isnt going to be json, its just the __repr__ of the the _dataset class